pkg/util/node.go

```diff
@@ -49,3 +56,28 @@ func IsNodeAddressTypeSupported(addrType corev1.NodeAddressType) bool {
 	}
 	return false
 }
+
+func GetReservedByNodeAnno(anno map[string]string) corev1.ResourceList {
```

Comment:
should move to the `apis/extension/node_reservation.go`

Reply:
As the cpuset package is used here, it has been suggested that the cpuset package should not be referenced in the api package.

Reply:
Agree with your point of view. Is it more appropriate to change the function name to GetNodeReservationFromAnnotation?